README.md

```diff
@@ -3,40 +3,42 @@
 ## Setup the dev env
 ### Windows
 1. Install [vagrant](https://www.vagrantup.com/docs/installation/) for your OS. Disable the hyper-v from [here](https://ugetfix.com/ask/how-to-disable-hyper-v-in-windows-10/).
-2. Install Git Bash (bundled with git) from [here](https://git-scm.com/downloads).
-3. Enable VT-x from your BIOS.
-4. Run Git Bash as administrator and run
+2. Install latest [VirtuaBox](https://www.virtualbox.org/wiki/Downloads) for your windows system. This is **required**.
+3. Install Git Bash (bundled with git) from [here](https://git-scm.com/downloads).
+4. Enable VT-x from your BIOS.
+5. Run Git Bash as administrator and run
     ```
     git config --global core.symlinks true
     ```
-5. Confirm the setting
+6. Confirm the setting
     ```
     git config core.symlinks
     ```
-    If you see true, you can go ahead.
-6. Clone this repository anywhere on your PC
+    If you see `true`, you can go ahead.
+7. Clone this repository anywhere on your PC
     ```bash
     git clone "https://github.com/Zeal-Student-Developers/issue-tracker.git"
     ```
-7. Run the following commands
+8. Run the following commands
     ```bash
     cd issue-tracker
     vagrant plugin install vagrant-vbguest
-    vagrant up
+    vagrant up --provider=virtualbox
     ```
-    First time it will run, it will take some time to download and setup the vm, and then provision it.
-8. Once vagrant is up and running, you need to connect to it
+    First time it will run, it will take some time to download and setup the vm, and then provision it. 
+    > Note: You may get *timed out* error and will probably need to run this command a several times to get it working. It's an issue with vagrant.
+9. Once vagrant is up and running, you need to connect to it
     ```bash
     vagrant ssh
     ```
-9. Run the following commands to complete the provisioning which was started earlier. This will setup the volumes required for node_modules and mongodb database. Also it will setup the containers for nodejs and mongodb.
+10. Run the following commands to complete the provisioning which was started earlier. This will setup the volumes required for node_modules and mongodb database. Also it will setup the containers for nodejs and mongodb.
     ```
     tools/setup/setup full
```

Comment:
Don’t we need to run tools/setup/provision first before tools/setup/setup full.

Reply:
`tools/setup/provision` is automatically run, by the `Vagrantfile` on the first `vagrant up` command. So no need to run it manually.

Reply:
In my case it was needed to run manually

Reply:
Sometimes when you have problems running vagrant for the first time, and if it gets interrupted the provision script doesn't run. So we have to manually do that.